recipes/sundials/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sundials" %}
+{% set version = "2.6.2" %}
+{% set sha256 = "d8ed0151509dd2b0f317b318a4175f8b95a174340fc3080b8c20617da8aa4d2f" %}
+{% set md5 = "3deeb0ede9f514184c6bd83ecab77d95" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://pkgs.fedoraproject.org/repo/pkgs/sundials/sundials-2.6.2.tar.gz/{{ md5 }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - blas_{{ variant }}  # [not win]
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+
+requirements:
+  build:
+    - cmake
+    - blas 1.1 {{ variant }}  # [not win]
+    - openblas 0.2.18*  # [not win]
+  run:
+    - blas 1.1 {{ variant }}  # [not win]
+    - openblas 0.2.18*  # [not win]
+
+test:
+  requires:
+    - cmake
```

Comment:
This is another annoying caveat of the features - since this environment contains only `cmake`, conda doesn't know which feature to activate. So you have to put Python as a requirement for building and testing only on Windows.